packages/analytics/src/utils/getters.ts

```diff
@@ -126,3 +126,32 @@ export const getCookie = (name: string): string | void => {
     return parts.pop()?.split(';').shift();
   }
 };
+
+/**
+ * Obtain checkout order generic fields from analytics properties.
+ *
+ * @param dataProperties - The event's properties data.
+ * @param orderIdFieldName - The name of orderId field. Default as `checkoutOrderId`.
+ * @returns The checkout generic order data result.
+ */
+export const getCheckoutOrderGenericProperties = (
+  dataProperties: EventProperties,
+  orderIdFieldName = 'checkoutOrderId',
+) => {
+  // Valid order code should be an alphanumeric value.
+  // Should be used 'orderId' values like e.g.: 5H5QYB
+  // and for 'checkoutOrderId' values like e.g.:123123123
+  const validOrderCode = isNaN(Number(dataProperties?.orderId));
```

Comment:
Small nitpick, since this is a boolean value, I'd rename it to `isOrderCodeValid` to ease reading a bit 🙂

Reply:
It was replaced, so It won't be needed anymore.